apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,134 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  IconButton,
+  NotificationMarker,
+  NotificationMarkerProps,
+} from '@itwin/itwinui-react';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  component: NotificationMarker,
+  args: {
+    enabled: true,
+    pulsing: false,
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    status: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          gap: '20px',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker {...args}>{args.children}</NotificationMarker>
+      </IconButton>
+      <NotificationMarker {...args}>3 members online</NotificationMarker>
+    </>
+  );
+};
+
+Basic.args = {
+  status: 'primary',
+  children: <SvgNotification />,
+};
+
+Basic.argTypes = {
+  children: {
+    control: false,
+  },
+};
+
+export const Pulsing: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='primary' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+
+      <NotificationMarker status='primary' {...args}>
+        <Text style={{ fontWeight: '700' }}>URGENT</Text>
+      </NotificationMarker>
+    </>
+  );
+};
+
+Pulsing.args = {
+  status: 'negative',
+  pulsing: true,
+};
+
+export const Status: Story<NotificationMarkerProps> = (args) => {
```

Comment:
Is this story showing too much information? Is there a better way? Or this is alright?

Basically I intended to display all four statuses for text and icons at once, similar to how we show all things at once for most components on the CSS repo's Github demo website.

Reply:
I like the story for the statuses. I think you don't need to show both text and icon. You could do something like this: 
![image](https://user-images.githubusercontent.com/37936169/191810190-e402e962-fbc3-45d1-b917-2ca4d84613e1.png)
That tells users also what the name of the color is.

Reply:
Thanks for the idea. I added the status names as the dummy names for the text notification examples for the `Status` story. 

However, I kept the icons examples in that story too since I felt it was worth showing both icons and texts. The user can then decide whether they want to use an icon, text, or something else.